theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -36,10 +36,12 @@ let fundingAgenciesApp = new Vue({
     crossreffApiDown: false, // if we are having trouble reaching the crossref api
     removeCharsFromQuery: ["."], // characters to be removed from search
     filteredWords: [],
+    ROR_API_URL: "https://api.ror.org/v2/organizations"
   },
   mounted() {
-    if (this.resourceMode === 'Edit' && (this.selfAccessLevel === 'owner' || this.selfAccessLevel === 'editor') && this.resPublished) {
-      this.checkFunderNamesExistInCrossref(this.fundingAgencies);
+    if (this.resourceMode === 'Edit' && (this.selfAccessLevel === 'owner' || this.selfAccessLevel === 'editor') ) {
```

Comment:
why did you remove the condition for `this.resPublished` here?

Reply:
I removed the this.resPublished condition because I wanted to allow funder name checks even before the resource is published. This helps catch any issues earlier. But, after doing some tests in Beta version, I thought it's better to keep it the this.resPublished condition.